src/WebJobs.Script.WebHost/Models/HostLogEntry.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Diagnostics;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.Models
+{
+    public class HostLogEntry
+    {
+        /// <summary>
+        /// Gets or sets the log level.
+        /// </summary>
+        [JsonConverter(typeof(StringEnumConverter))]
+        public TraceLevel Level { get; set; }
+
```

Comment:
If we have the ability to pass the different properties down to the logging infrastructure (so they go into the appropriate columns), it would be nice to have an optional FunctionName here.